x-pack/packages/kbn-cloud-security-posture-common/constants.ts

```diff
@@ -9,7 +9,7 @@ export const CSPM_POLICY_TEMPLATE = 'cspm';
 export const CDR_LATEST_NATIVE_MISCONFIGURATIONS_INDEX_PATTERN =
   'logs-cloud_security_posture.findings_latest-default';
 export const CDR_LATEST_THIRD_PARTY_MISCONFIGURATIONS_INDEX_PATTERN =
-  'logs-*_latest_misconfigurations_cdr';
+  'security_solution-*.misconfigurations_latest-*';
```

Comment:
Update index pattern according to: https://github.com/elastic/security-team/issues/10224